File: src/cpp/stickerrenderer.cpp
#include "stickerrenderer.h"

#include "glgraphics.h"

StickerRenderer::StickerRenderer()
{
}

void StickerRenderer::render(const Sticker &sticker) const
{
	vec2f stickerCenter = sticker.getCenter();
	Color stickerColor = sticker.getColor();

	GLGraphics *glGraphics = GLGraphics::getInstance();

	glGraphics->setColor(stickerColor);
	glGraphics->drawCircle(stickerCenter, 0.04, true);
}

File: src/js/stickerrenderer.js
class StickerRenderer
{
	constructor(graphics)
	{
		this.graphics = graphics
	}

	function render(sticker)
	{
		var stickerColor = sticker.getColor();
		var stickerCenter = sticker.getCenter();
		var radius = 16;

		this.graphics.setColor(stickerColor);
		this.graphics.drawCircle(stickerCenter, radius, true);
	}
}
